src/utilities/codemirror/codemirror-utils.ts

```diff
@@ -107,6 +108,48 @@ export function getMissingArgDefs(argInfoArray: ArgTextDef[]): FswCommandArgumen
     .map(argInfo => argInfo.argDef);
 }
 
+export function getDefaultVariableArgs(parameters: VariableDeclaration[]): string[] {
```

Comment:
Can this function have some unit tests, please?

Reply:
Sure, I add a unit test